lms/djangoapps/verify_student/views.py

```diff
@@ -1006,6 +1007,21 @@ def results_callback(request):
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
+        if checkpoints:
+            course_key = checkpoints[0].course_id
+            credit_requirement = get_credit_requirement(
+                course_key, "reverification", checkpoints[0].checkpoint_name
+            )
+            if credit_requirement is not None:
+                try:
+                    set_credit_requirement_status(
+                        attempt.user.username, credit_requirement, "satisfied"
+                    )
+                except Exception:  # pylint: disable=broad-except
+                    # Catch exception if unable to add credit requirement status for user
```

Comment:
Please try to make docstrings length (break if necessary) upto 79-80 as mentioned in this wiki https://github.com/edx/edx-platform/wiki/Python-Guidelines#follow-pep-8